sdk/keyvault/azure-keyvault-certificates/tests/test_certificates_models.py

```diff
@@ -0,0 +1,16 @@
+from azure.keyvault.certificates import CertificatePolicy
+
+def test_policy_expected_errors():
+    try:
+        cert_policy = CertificatePolicy("issuer-name", "subject-name")
+        assert False, "Error should've thrown since we only have one positional parameter"
+    except TypeError as e:
+        if '__init__' not in str(e):
+            raise
+
+    try:
+        cert_policy = CertificatePolicy("issuer-name")
+        assert False, "Error should be thrown since we haven't set a subject_name or sans"
+    except ValueError as e:
```

Comment:
FYI, `pytest.raises` is made for this sort of thing: https://github.com/Azure/azure-sdk-for-python/blob/382994348b289d4cb90dc74e2ddc9fadc7c5824d/sdk/identity/azure-identity/tests/test_identity.py#L325-L327

Reply:
switched to that, it looks a lot cleaner now, thanks!